config/certification-settings.js

```diff
@@ -75,6 +75,27 @@ const superBlockCertTypeMap = {
   'machine-learning-with-python-v7': certTypes.machineLearningPyV7
 };
 
+const superBlockCertTypeMap = {
```

Comment:
Is this still necessary? I don't see it being used anywhere.

Reply:
I plan on using this, in #41147. Otherwise, I think I should be using it in this PR... will need to double check.